stories/form-group/form-group.stories.js

```diff
@@ -36,3 +36,125 @@ export const required = () => `<div class="fd-form-group">
+                </div>
+                <div class="fd-col fd-col-md--2 fd-col-lg--4 fd-col-xl--12">
+                <label class="fd-form-label" for="input-14y-name">Name:</label>
+                </div>
+                <div class="fd-col fd-col-md--10 fd-col-lg--8 fd-col-xl--12">
+                <input class="fd-input" type="text" id="input-14y-name" placeholder="Enter First and Last Name" value="Amelia Perry">
+                </div>
+            </div>
+        </div>
+
+        <div class="fd-form-group fd-col fd-col-md--6 fd-col-lg--6 fd-col-xl--6 fd-col--wrap">
+            <div class="fd-form-group__header">
+                <span class="fd-form-group__header-text">Group Header 2</span>
+            </div>
+            <div class="fd-row">
+                <div class="fd-col fd-col-md--2 fd-col-lg--4 fd-col-xl--12">
+                <label class="fd-form-label" for="input-14a-name">Name:</label>
+                </div>
+                <div class="fd-col fd-col-md--10 fd-col-lg--8 fd-col-xl--12">
+                <input class="fd-input" type="text" id="input-14a-name" placeholder="Enter First and Last Name" value="Amelia Perry">
+                </div>
+                <div class="fd-col fd-col-md--2 fd-col-lg--4 fd-col-xl--12">
+                <label class="fd-form-label" for="input-14b-name">Name:</label>
+                </div>
+                <div class="fd-col fd-col-md--10 fd-col-lg--8 fd-col-xl--12">
+                <input class="fd-input" type="text" id="input-14b-name" placeholder="Enter First and Last Name" value="Amelia Perry">
+                </div>
+                <div class="fd-col fd-col-md--2 fd-col-lg--4 fd-col-xl--12">
+                <label class="fd-form-label" for="input-14c-name">Name:</label>
+                </div>
+                <div class="fd-col fd-col-md--10 fd-col-lg--8 fd-col-xl--12">
+                <input class="fd-input" type="text" id="input-14c-name" placeholder="Enter First and Last Name" value="Amelia Perry">
+                </div>
+            </div>
+        </div>
+        <div class="fd-form-group fd-col fd-col-md--6 fd-col-lg--6 fd-col-xl--6 fd-col--wrap">
+            <div class="fd-form-group__header">
+                <span class="fd-form-group__header-text">Group Header 3</span>
+            </div>
+            <div class="fd-row">
+                <div class="fd-col fd-col-md--2 fd-col-lg--4 fd-col-xl--12">
+                <label class="fd-form-label" for="input-14j-name">Name:</label>
+                </div>
+                <div class="fd-col fd-col-md--10 fd-col-lg--8 fd-col-xl--12">
+                <input class="fd-input" type="text" id="input-14j-name" placeholder="Enter First and Last Name" value="Amelia Perry">
+                </div>
+                <div class="fd-col fd-col-md--2 fd-col-lg--4 fd-col-xl--12">
+                <label class="fd-form-label" for="input-14h-name">Name:</label>
```

Comment:
I'm not sure if keeping label and input in separate grid columns is a good idea. You need to make double work to maintain responsive grid. In my opinion this is quite unusual approach bringing to value but more code into the HTML.

Reply:
I am just following the existing Form Grid example here and have not modified any underlying logic for responsive grid column usage.